query-graphs-js/src/graphPath.ts

```diff
@@ -1633,6 +1671,7 @@ function advancePathWithDirectTransition<V extends Vertex>(
         // it uses @interfaceObject on an interface of that implementation.
         details = `cannot find implementation type "${fieldTypeName}" (supergraph interface "${path.tail.type.name}" is declared with @interfaceObject in "${subgraph}")`;
       } else {
+        // TODO: from sachin - this is the place to "pretend" the field doesn't exist
```

Comment:
You can remove this comment since it's handled above now.

Reply:
[2391f43](https://github.com/apollographql/federation/pull/2902/commits/2391f43bfcb9e698f38ee7da77dee9696076c82c)